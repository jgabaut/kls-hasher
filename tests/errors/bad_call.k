#!/bin/sh

./kls_hasher -bad
