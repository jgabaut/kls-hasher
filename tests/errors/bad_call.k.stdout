./kls_hasher v0.0.1
KLS API LVL:    {31400}
./kls_hasher reference date: {}
[DEBUG] File:    {-bad}
